salt/environment_settings.yml

```diff
@@ -19,6 +19,7 @@ edxapp_secret_backends: &edxapp_secret_backends
       - etl
       - grades
       - storage
+      - edx-video-upload
```

Comment:
We're adding this feature to residential as well?

Reply:
Not necessarily, but I figured it's easier to keep things similar in case we decide to port it over.